### ISSUE TITLE
Adding Quartz Variants in Config Causes Crash

### ISSUE THREAD

**Issue by iamc24** [PersonTheCat/ore_stone_variants#38]
I added biotite ore variants and quartz ore variants in the config, but when I launched the game, this mod caused a crash.
config file - [here](https://pastebin.com/pyZm4yML)
crash report - [here](https://pastebin.com/UHKZxkZA)

**Comment by PersonTheCat** [comment]
Okay, good I see. The actual name of biotite's properties is "quark_biotite_ore," not just "biotite_ore." Is there any way you could try changing that and let me know if the issue is resolved? If not, I may be able to look at it either tonight or tomorrow morning. Thanks.

**Comment by iamc24** [comment]
It doesn't work, it is still the same crash.

**Comment by iamc24** [comment]
I just did a little bit of experimenting and I think I've narrowed the problem down to quartz ore variants. Biotite variants with out quartz variants doesn't crash and works properly with all variants created. Quartz variants with out biotite variants doesn't work - it will always crash.

**Comment by PersonTheCat** [comment]
Hey, sorry it took me so long to get back to you. I originally never figured out what was wrong before deciding to completely rework the system, which took a super long time and is still sort of WIP. All of the mechanisms that might have caused this problem before are no longer really in place. If you're still looking for a fix, I was hoping you could try the new version at some point and let me know whether it continues. If it does, I'll definitely try to work something out. If something else goes wrong and you'd prefer to stick with 3.x, let me know and I'll create a new branch for 3.x to try and fix it that way.

**Comment by iamc24** [comment]
is the config the same? i would like to not break the world i have currently if the fix does work.
on another note, i have been busy lately so i might have to wait to test it until the weekend, if you can wait that long

**Comment by PersonTheCat** [comment]
It is supposed to be the same, but 2-3 different options have been changed because so much is handled by the block registry now. I've made some more changes since it was released to improve a couple of things, though, and it's definitely looking about ready at this point. Here's what will break from previous setups: 
*Properties added under /config/ore_stone_variants_mods/ are not automatically registered anymore.
*Entries that previously used an asterisk to retrieve all states of the same block will now be registered as one single block instead of individual blocks. 
*Model names have changed, but I'm not aware of anyone actually changing those via resource packs or anything.
